features/call/impl/src/main/kotlin/io/element/android/features/call/impl/ui/ElementCallActivity.kt

```diff
@@ -115,6 +120,11 @@ class ElementCallActivity :
         }
     }
 
+    private fun setCallIsActive() {
+        requestAudioFocus()
+        CallForegroundService.start(this)
```

Comment:
Don't we ever stop the foreground service?

Reply:
Yes, I kept the existing code in `onDestroy()`

Reply:
Oh, I didn't see it in the diff, so I assumed it wasn't there 🤦 . Thanks!